src/mono/msbuild/android/build/AndroidApp.InTree.targets

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <!-- This depends on the root Directory.Build.targets imported this file -->
+  <UsingTask TaskName="MonoAOTCompiler" AssemblyFile="$(MonoAOTCompilerTasksAssemblyPath)" />
+  <UsingTask TaskName="RuntimeConfigParserTask" AssemblyFile="$(MonoTargetsTasksAssemblyPath)" />
```

Comment:
There is a redundant UsingTask in tests.mobile.targets L70 https://github.com/dotnet/runtime/blob/f1dbe2ec06682a67615c8193a017ed2d6220fa84/eng/testing/tests.mobile.targets#L70

If all of the mobile intree targets will include these, would it make sense to have a common mobile intree targets?

Reply:
Yes, but I would prefer doing such a thing in a follow up.  With this change, Wasm, iOS, and Android are structured in a similar way (wasm has quite a bit more) where we can start to identify common targets.